pandas/_typing.py

```diff
@@ -32,6 +33,7 @@
 FilePathOrBuffer = Union[str, Path, IO[AnyStr]]
 
 FrameOrSeries = TypeVar("FrameOrSeries", bound="NDFrame")
+IndexOrSeries = TypeVar("IndexOrSeries", bound="IndexOpsMixin")
```

Comment:
is this meant. it to include array likes, eg EA?

Reply:
Nope, just Index and Series.